BibBuddy/app/src/main/java/de/bibbuddy/NoteRecyclerViewAdapter.java

```diff
@@ -67,81 +59,61 @@ public MyViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     Context context = parent.getContext();
     itemView =
         LayoutInflater.from(context).inflate(R.layout.list_view_item_note, parent, false);
-    background = itemView.getBackground();
     hiddenDeletePanel = parent.getRootView().findViewById(R.id.hidden_delete_panel);
     panelDelete = hiddenDeletePanel.findViewById(R.id.panel_delete);
     return new MyViewHolder(itemView);
```

Comment:
Newline for better readability.
```suggestion

    return new MyViewHolder(itemView);
```